repos/singlesession/github/kamidipreetham/soletta-test/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
The only repos folder that we will ship is the demo one that contains Samples from Soletta